python/ray/data/_internal/planner/plan_map_op.py

```diff
@@ -0,0 +1,64 @@
+from typing import Any, Iterator
+
+import ray
+from ray.data._internal.compute import ActorPoolStrategy, TaskPoolStrategy, get_compute
+from ray.data._internal.execution.interfaces import PhysicalOperator
+from ray.data._internal.execution.operators.map_operator import MapOperator
+from ray.data._internal.logical.operators.map_operator import AbstractMap
+from ray.data.block import Block, CallableClass
+
+
+def _plan_map_op(op: AbstractMap, input_physical_dag: PhysicalOperator) -> MapOperator:
+    """Get the corresponding physical operators DAG for AbstractMap operators.
+
+    Note this method only converts the given `op`, but not its input dependencies.
+    See Planner.plan() for more details.
+    """
+    compute = get_compute(op._compute)
+    block_fn = op._block_fn
```

Comment:
Should we also remove block_fn from the op?

Reply:
@ericl - yes, plan to do in a separate PR - https://github.com/ray-project/ray/pull/31977#discussion_r1089348501 .